### PR TITLE
Test schema health

### PR DESCRIPTION
**Pull request by Ladas**
Depends on:
- [x] https://github.com/ManageIQ/topological_inventory-core/pull/90

### CODE REVIEW

**Comment by agrare** [comment]
> Thoughts? Any voices against the gem above? The main motivation is that it's impossible to check these manually (even I am forgetting to add stuff) :-)

This is an awesome idea, haven't looked at the gem yet but I'm all for this.  Only issue I can think of is we might end up having to maintain a huge list of "overrides" for stuff that doesn't conform, especially when we try to merge with the MIQ schema.

**Comment by Ladas** [comment]
@agrare well I would say that if we are going to merge it with MIQ, the specs will have to pass without any overrides. :-) We should just fix the DB.

**Comment by agrare** [comment]
That's optimistic ;)

File: db/migrate/20190124154742_add_missing_not_null_constraints_to_source_ref.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class AddMissingNotNullConstraintsToSourceRef < ActiveRecord::Migration[5.2]
+  def change
+    change_column_null(:container_images, :source_ref, false)
```
Comment (agrare) [inline db/migrate/20190124154742_add_missing_not_null_constraints_to_source_ref.rb]: Don't worry about the parens just to appease rubocop.  We just need to add this method to the overrides.

Comment (Ladas) [reply db/migrate/20190124154742_add_missing_not_null_constraints_to_source_ref.rb]: hm ok

Comment (bdunne) [reply db/migrate/20190124154742_add_missing_not_null_constraints_to_source_ref.rb]: https://github.com/ManageIQ/guides/pull/335

**Comment by bdunne** [comment]
:+1: Sorry for the delay, this looks great

**Pull request merged**